firefox-ios/firefox-ios-tests/Tests/XCUITests/PhotonActionSheetTests.swift

```diff
@@ -81,14 +90,13 @@ class PhotonActionSheetTests: BaseTestCase {
     // https://testrail.stage.mozaws.net/index.php?/cases/view/2306841
     // Smoketest
     func testSharePageWithShareSheetOptions() {
-        // Temporarily disabled until url bar redesign work FXIOS-8172
-//        openNewShareSheet()
-//        mozWaitForElementToExist(app.staticTexts["Open in Firefox"], timeout: 10)
-//        XCTAssertTrue(app.staticTexts["Open in Firefox"].exists)
-//        XCTAssertTrue(app.staticTexts["Load in Background"].exists)
-//        XCTAssertTrue(app.staticTexts["Bookmark This Page"].exists)
-//        XCTAssertTrue(app.staticTexts["Add to Reading List"].exists)
-//        XCTAssertTrue(app.staticTexts["Send to Device"].exists)
+        openNewShareSheet()
```

Comment:
If we are testing if the "share" options are available, we can launch the screen from the hamburger menu. See the updates from `openNewShareSheet()`.